components/brave_rewards/browser/publisher_info_database_unittest.cc

```diff
@@ -835,7 +836,7 @@ TEST_F(PublisherInfoDatabaseTest, GetActivityList) {
   filter_1.excluded = ledger::EXCLUDE_FILTER::FILTER_ALL;
   EXPECT_TRUE(publisher_info_database_->GetActivityList(0,
                                                         0,
-                                                        filter_1,
+                                                        filter_1.Clone(),
```

Comment:
`filter_1.Clone()` should use `std::move(filter_1)` as `filter_1` is no longer used